backend/s3/s3.go

```diff
@@ -1381,6 +1393,19 @@ func (o *Object) split() (bucket, bucketPath string) {
 	return o.fs.split(o.remote)
 }
 
+// getClient makes an http client according to the options
+func getClient(opt *Options) *http.Client {
+	// TODO: Do we need cookies too?
+	t := fshttp.NewTransportCustom(fs.Config, func(t *http.Transport) {
+		if opt.DisableHTTP2 {
```

Comment:
Aside: It occurs to me that this would be easy to make a global option... That doesn't disable it per backend but it might be useful for debugging.

Reply:
Yep, had the same thought. I also wanted to add experimental support for quic (using quic-go) in the fshttp client so will probably do both of them together when I get around to it.

Reply:
Great! That would be very useful.